backstop/tests/column-filter.html

```diff
@@ -0,0 +1,221 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html id="theme">
+  <head>
+    <title>Column filter | iTwinUI</title>
+    <link
+      rel="icon"
+      href="../assets/favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/global.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/button.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/inputs.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/toggle-switch.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/table.css"
```

Comment:
I am just thinking now... If column filter is going to be a separate component, we might want to import only filtering css, not whole table css. @bentleyvk thoughts?

Reply:
It kind of makes sense to have everything table related under table CSS. But it takes a huge table CSS file. However if you are using table those rules are going to be imported anyway 😀 I wonder if anyone wants to use filters outside the table. But maybe we can still have it as a separate component as it already has its own test file.